setup.py

```diff
@@ -15,17 +15,21 @@
     packages=setuptools.find_packages(),
     classifiers=[
         "Programming Language :: Python :: 3",
-        "Development Status :: 3 - Alpha", "Intended Audience :: Developers",
+        "Development Status :: 3 - Alpha",
+        "Intended Audience :: Developers",
         "Intended Audience :: System Administrators",
         "License :: OSI Approved :: BSD License",
         "Operating System :: MacOS :: MacOS X",
-        "Operating System :: POSIX :: Linux", "Topic :: System :: Clustering",
+        "Operating System :: POSIX :: Linux",
+        "Topic :: System :: Clustering",
         "Topic :: System :: Distributed Computing",
         "Topic :: System :: Installation/Setup",
         "Topic :: System :: Systems Administration",
-        "Topic :: System :: Networking"
+        "Topic :: System :: Networking",
+    ],
+    install_requires=[
+        "psutil",
```

Comment:
One option to install the dependencies is creating a virtual environment after downloading nearup, and then installing all dependencies local to nearup.

Reply:
Yes, we could add a requirements.txt file in that case and parse it here, as well so we give people an option of one or the other.

Reply:
Realized this isn't enough, we never documented people should install and run nearup via pip (and i explicitly remove main.py from setup.py. therefore, all people are using installation scripts. And nearup comes with auto update, means you must pack psutil or make `rootdir/nearup` script create venv, install psutil and rerun nearup in next update otherwise **All people run nearup will break when we do next betanet release**